packages/terra-framework-docs/src/terra-dev-site/test/table/mockSortableTableData.json

```diff
@@ -1,8 +1,8 @@
 {
   "cols": [
-    { "id": "Column-0", "displayName": "Vitals", "isSelectable": true, "sortIndicator": "ascending" },
-    { "id": "Column-1", "displayName": "March 16", "isSelectable": true },
-    { "id": "Column-2", "displayName": "March 17" },
+    { "id": "Column-0", "displayName": "Vitals", "isSelectable": true, "sortIndicator": "ascending", "isResizable": true },
+    { "id": "Column-1", "displayName": "March 16", "isSelectable": true, "isResizable": true },
+    { "id": "Column-2", "displayName": "March 17", "isSelectable": true },
```

Comment:
Why did this need to change for the Terra Table?

Reply:
This was just to provide a way for people to test column resizing and selectable columns in table.